### DISCUSSION TITLE
Multiple mind files for individual tracker images?

### DISCUSSION THREAD

**Discussion by mike-shields-dev** [hiukim/mind-ar-js#137] [Q&A]
Hi there, 

I'm just starting to investigate this library, I have checked out the examples in the documentation regarding multiple targets. My understanding is that multiple tracker images are compiled to a single .mind file which is attributed to the ```<a-scene>``` element and each child ```<a-entity>``` element then refers to an indexed image within the .mind file? 

However, I would like to use this library with React so that I can update/add/remove assets (tracker images and 3d models) on a backend API and have React render the neccesary A-Frame components. 

Is there a way to use multiple .mind files, one for each tracking image?

**Comment by kanzaky** [comment]
I´ve that same problem here
